.github/workflows/ci.yaml

```diff
@@ -56,9 +72,57 @@ jobs:
     with:
       charm-path: ./charms/${{ matrix.charm }}
```

Comment:
suggestion(non-blocking): same as check libs: this job can be updated to use `get-charm-paths` output to populate the matrix instead of hard-coding it

Reply:
Done!

9d02544